minio/api.py

```diff
@@ -1191,7 +1192,8 @@ def list_incomplete_uploads(self, bucket_name, prefix=None,
 
         return self._list_incomplete_uploads(bucket_name, prefix, recursive)
 
-    def _list_incomplete_uploads(self, bucket_name, prefix=None, recursive=False, is_aggregate_size=True):
+    def _list_incomplete_uploads(self, bucket_name, prefix=None,
```

Comment:
All these changes are unrelated .. lets not do formatting in a change which has nothing to do with it.